packages/broker/test/unit/plugins/legacyWebsocket/Connection.test.ts

```diff
@@ -1,33 +1,82 @@
 import { Connection } from '../../../../src/plugins/legacyWebsocket/Connection'
 import { Stream } from '../../../../src/Stream'
-import { Todo } from '../../../../src/types'
+import { EventEmitter } from "events"
+import WebSocket from "ws"
+import { Protocol } from "streamr-network"
+import { waitForCondition, waitForEvent } from 'streamr-test-utils'
+import Mock = jest.Mock
+import { ErrorResponse } from '../../../../../protocol/src'
```

Comment:
This should import from `streamr-network` package instead of relative monorepo folder

Reply:
fixed